manager-bundle/src/Command/MaintenanceModeCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Error\SyntaxError;
+use Webmozart\PathUtil\Path;
+
+/**
+ * @internal
+ */
+class MaintenanceModeCommand extends Command
+{
+    private string $webDir;
+    private Environment $twig;
+    private Filesystem $filesystem;
+
+    public function __construct(string $webDir, Environment $twig, Filesystem $filesystem = null)
+    {
+        $this->webDir = $webDir;
+        $this->twig = $twig;
+        $this->filesystem = $filesystem ?? new Filesystem();
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('contao:maintenance-mode')
+            ->addArgument('state', InputArgument::REQUIRED, 'Use "enable" to enable and "disable" to disable the maintenance mode. If the state is already the desired one, nothing happens. You can also use "on" and "off".')
+            ->addOption('template', 't', InputOption::VALUE_REQUIRED, 'Allows to take a different Twig template name when enabling the maintenance mode.', '@ContaoCore/Error/service_unavailable.html.twig')
+            ->addOption('templateVars', null, InputOption::VALUE_OPTIONAL, 'Add custom template variables to the Twig template when enabling the maintenance mode (provide as JSON).', '{}')
+            ->setDescription('Changes the state of the system maintenance mode. Without any argument, it toggles between the current state.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $shouldEnable = \in_array($input->getArgument('state'), ['enable', 'on'], true);
```

Comment:
What about using `filter_var`? This handles probably all "boolean config" cases.

```suggestion
        $shouldEnable = filter_var($input->getArgument('state'), FILTER_VALIDATE_BOOLEAN);
```

Reply:
I would prefer keeping the existing code because it has to match the command docs. Otherwise we have to document that `1` and `yes` work as well. I still hope that Symfony Console gets support for enums at some point so the options would be predefined anyway.